### ISSUE TITLE
Call stack frames should be keyboard accessible

### ISSUE THREAD

**Issue by jasonLaster** [replayio/devtools#5208]
**Description:**
I should be able to navigate up and down the call stack with arrow up / arrow down

**Comment by jonbell-lot23** [comment]
Going through UI polish tasks and adding comments:

I'd like to see us move this out of icebox. It's the sort of thing that we'll keep punting if don't make a concerted effort, and keyboard command are extremely important for people staying in flow.

**Comment by gideonred** [comment]
This ticket has been moved to [FE-126](https://linear.app/replay/issue/FE-126/call-stack-frames-should-be-keyboard-accessible)
